lib/src/geo/crs/crs.dart

```diff
@@ -350,21 +350,21 @@ abstract class Projection {
 
   LatLng unproject(CustomPoint point);
 
-  double _inclusive(Comparable start, Comparable end, double value) {
-    if (value.compareTo(start as num) < 0) return start as double;
-    if (value.compareTo(end as num) > 0) return end as double;
+  double _inclusive(double start, double end, double value) {
+    if (value.compareTo(start) < 0) return start;
```

Comment:
Do we really need to keep using the `compareTo` method now that they are all doubles?